scripts/lib/CIME/check_lockedfiles.py

```diff
@@ -77,33 +77,34 @@ def check_pelayouts_require_rebuild(case, models):
 
         unlock_file("env_mach_pes.xml", case.get_value("CASEROOT"))
 
-def check_lockedfiles(caseroot=None):
+def check_lockedfiles(case):
```

Comment:
It seems that this change is independent of this PR. If this is the case - it should be brought in as a separate PR. If this is not the case - why did this change to the interface need to be part of this PR.